include/api/CDataFrameAnalysisRunner.h

```diff
@@ -148,6 +148,9 @@ class API_EXPORT CDataFrameAnalysisRunner {
     //! \return Callback function for writing state using given persist inserter
     TStatePersister statePersister();
 
+protected:
```

Comment:
Do we need this? There is already a protected `spec()` function that returns it. If we do need it for some reason then should we remove the function?

Reply:
Done